stock_no_negative/i18n/fr.po

```diff
@@ -96,24 +91,10 @@ msgstr "Quants"
 #, python-format
 msgid ""
 "You cannot validate this stock operation because the stock level of the "
-"product '{name}'{name_lot} would become negative ({q_quantity}) on the stock "
-"location '{complete_name}' and negative stock is not allowed for this "
+"product '{name}'{name_lot} would become negative ({q_quantity}) on the stock"
+" location '{complete_name}' and negative stock is not allowed for this "
 "product and/or location."
 msgstr ""
-
-#, python-format
-#~ msgid " lot '%s'"
-#~ msgstr " lot '%s'"
-
-#~ msgid "Product Template"
-#~ msgstr "Modèle d'article"
-
-#, python-format
-#~ msgid ""
-#~ "You cannot validate this stock operation because the stock level of the "
-#~ "product '%s'%s would become negative (%s) on the stock location '%s' and "
-#~ "negative stock is not allowed for this product and/or location."
-#~ msgstr ""
-#~ "Impossible de valider cette opération car le niveau de stock de ce "
-#~ "produit %s'%s deviendrait négative(%s) dans l'emplacement du stock '%s'. "
-#~ "Un stock négatif n'est pas permis pour ce produit."
+"Impossible de valider cette opération car le niveau de stock de ce "
+"produit %s'%s deviendrait négative(%s) dans l'emplacement du stock '%s'. "
```

Comment:
le niveau de stock... négati**f**

Reply:
Hi @rousseldenis is there anything wrong with this translation ? Do I need to keep it as it is the way it's commented in v16 ?

Reply:
Why have you changed this string ?

Reply:
> Why have you changed this string ?

Hi @rousseldenis, haven't changed those strings , I just made those 3 lines of translation uncommented , should I keep it commented ?

Reply:
Ok, didn't see. Could you replace 'négative' by 'négatif' ? Thanks